tests/integration/kubeflags/kubeflags_test.go

```diff
@@ -108,10 +108,88 @@ var _ = Describe("create a new cluster with kube-* flags", Ordered, func() {
 					return nil
 				}
 				return errors.New("error finding kube-proxy")
-			}, "120s", "15s").Should(Succeed())
+			}, "300s", "15s").Should(Succeed())
+		})
+		It("kills previous server, clean logs and start up with no problems and disabled kube-proxy and cloud controller", func() {
```

Comment:
Create a new `When` Block for this entire subtest
```suggestion
            When("server is setup without cloud controller components")
		It("kills previous server, clean logs and start up with no problems and disabled kube-proxy and cloud controller", func() {
```

Also, seperate the `Kill server and cleanup` and put the `It` in the previous section.